doc/examples/segment_quickbundles.py

```diff
@@ -36,10 +37,13 @@
 threshold. Keep in mind that since the MDF metric requires streamlines to have
 the same number of points, the clustering algorithm will internally use a
 representation of streamlines that have been automatically downsampled/upsampled
-so they have only 18 points (See tutorial### to manually set that number of points).
+so they have only 12 points (to manually set the number of points one can use
+the `ResampleFeature`).
 """
 
-qb = QuickBundles(threshold=10.)
+feature = ResampleFeature(nb_points=12)
+metric = AveragePointwiseEuclideanMetric(feature=feature)  # a.k.a. MDF
```

Comment:
Regarding the comment at the end of this line: wouldn't it be better to change the references above to `MDF` to talk about this metric instead? It's not like 'MDF' is a very commonly used acronym.